src/components/TodoNotification/TodoNotification.tsx

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import classNames from 'classnames';
+
+import React from 'react';
+import { useTodo } from '../TodoContext/TodoContext';
+
+export const TodoNotification: React.FC = () => {
+  const {
+    isError,
+    setIsError,
+    errorMessage,
+    setErrorMessage,
+  } = useTodo();
+
+  const handleButtonClick = () => {
+    setIsError(false);
+    setErrorMessage('');
+  };
+
+  return (
+
+    <div className={classNames(
+      'notification is-danger is-light has-text-weight-normal',
+      { hidden: !isError },
+    )}
+    >
+      <button
+        type="button"
+        className="delete"
+        title="Click here"
+        onClick={handleButtonClick}
+      />
+
+      {errorMessage}
+    </div>
+  );
+};
+
+// {/* show only one message at a time */ }
+//       Unable to add a todo
+//   < br />
+//   Unable to delete a todo
+//     < br />
+//     Unable to update a todo
```

Comment:
Remove comments. Try it write obvious code that explain all we need without comments)